notebooks/stress-tests.ipynb

```diff
@@ -38,8 +38,8 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    for execution in range(3):
</pre><p>Same configurable comment for this range(3), not harcode the 3 but make it configurable. Is that 3 the same as the n_threads = 3?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
No, this is not the same.
For the THREDDS stress test, I wanted to test many factors:
- Number of runs
- Number of threads
- Number of executions

This way we can create different scenarios :
- 1 executions of 3 threads of 300 runs : Resulting in 900 requests
- 2 executions of 1 threads of 300 runs : Resulting in 600 requests
- 2 executions of 3 threads of 100 runs : Resulting in 600 requests

But at the end I removed the first loop for the number of executions because it seems to me that it is the same as increasing the amount of request (runs) and the number of threads.